packages/headless/src/integration-tests/analytics-migration.test.ts

```diff
@@ -258,6 +368,281 @@ describe('Analytics Search Migration', () => {
     assertNextEqualsLegacy(callSpy);
   });
 
+  it('analytics/dateFacet/breadcrumb', async () => {
+    legacySearchEngine.addReducers({
+      dateFacetSet: dateFacetSetReducer,
+    });
+    nextSearchEngine.addReducers({
+      dateFacetSet: dateFacetSetReducer,
+    });
+    legacySearchEngine.dispatch(
+      registerDateFacet({
+        facetId: ANY_FACET_ID,
+        field: ANY_FACET_ID,
+        generateAutomaticRanges: true,
+      })
+    );
+    nextSearchEngine.dispatch(
+      registerDateFacet({
+        facetId: ANY_FACET_ID,
+        field: ANY_FACET_ID,
+        generateAutomaticRanges: true,
+      })
+    );
+    const action = executeSearch({
+      legacy: logDateFacetBreadcrumb({
+        facetId: ANY_FACET_ID,
+        selection: ANY_RANGE_FACET_BREADCRUMB_VALUE,
+      }),
+      next: {
+        actionCause: SearchPageEvents.breadcrumbFacet,
+        getEventExtraPayload: (state) =>
+          new SearchAnalyticsProvider(
+            () => state
+          ).getRangeFacetBreadcrumbMetadata(
+            ANY_FACET_ID,
+            ANY_RANGE_FACET_BREADCRUMB_VALUE
+          ),
+      },
+    });
+
+    legacySearchEngine.dispatch(action);
+    nextSearchEngine.dispatch(action);
+    await wait();
+
+    assertNextEqualsLegacy(callSpy);
+  });
```

Comment:
Weird case where the state absolutely had to have facets for the analytics to work. I added them manually with actions.